drivers/hmis/lib/form_data/static/external_form_submission_review.json

```diff
@@ -0,0 +1,31 @@
+{
+  "item": [
+    {
+      "type": "TEXT",
+      "link_id": "notes",
+      "text": "User Notes",
+      "mapping": {
+        "field_name": "notes"
+      },
+      "required": true
+    },
+    {
+      "link_id": "reviewed",
```

Comment:
I wasn't able to find: is there an existing pattern in our form json definitions matching this image from the mocks,
<img width="270" alt="Screenshot 2024-03-08 at 3 12 25 PM" src="https://github.com/greenriver/hmis-warehouse/assets/16002775/e0e429b9-2ecd-429e-a444-b64c92da877d">

? I thought of putting them in an item group with only one item in it, but it looks very different visually of course.

Reply:
I don't know that there is a way to replicate this treatment right now. But it seems functional as-is so probably okay for now.

Reply:
Ok, sounds good! I updated both checkboxes to use the word 'mark' for consistency.